### PR TITLE
chore: update website for macOS airgap binaries

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
the airgap installer is no longer 'universal' as it was too big (reach 2GB limit max of GitHub artifacts)

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/src/pages/downloads/macos.tsx [hunk 1]
```diff
@@ -48,13 +52,32 @@ async function grabfilenameforMac(
...
+  const airgapsetupX64 = macosX64AirgapSetupAssets?.[0]?.browser_download_url;
+
+  const macosArm64AirgapSetupAssets = assets.filter(
+    asset => (asset.name as string).endsWith('-arm64.dmg') && asset.name.includes('airgap'),
+  );
+
+  const airgapsetupArm64 = macosArm64AirgapSetupAssets?.[0]?.browser_download_url;
+
+  console.log('airgapsetupX64', airgapsetupX64);
+  console.log('airgapsetupArm64', airgapsetupArm64);
```
Comment (feloy) [inline website/src/pages/downloads/macos.tsx]: to be removed?

**Review by feloy — APPROVED** [review]
tested, by fetching the release info for v1.10.1 instead of current `latest`, works fine, I can download the 4 different DMG files

File: website/src/pages/downloads/macos.tsx (hunk 1 above)
Comment (benoitf) [reply website/src/pages/downloads/macos.tsx]: fixed, thanks

**Review by deboer-tim — APPROVED** [review]
Verified against v1.10.2.

**Pull request merged**